examples/text-generation/utils.py

```diff
@@ -451,9 +454,7 @@ def setup_distributed_model(args, model_dtype, model_kwargs, logger):
         # Construct model with fake meta tensors, later will be replaced on devices during ds-inference ckpt load
         with deepspeed.OnDevice(dtype=model_dtype, device="meta"):
             if (
-                hasattr(config, "rope_scaling")
-                and config.rope_scaling
-                and config.rope_scaling["rope_type"] == "llama3"
+                any(model_type in args.model_name_or_path.lower() for model_type in MODELS_WITH_POS_EMBEDDING_LIMIT)
```

Comment:
```suggestion
                any(model_type == config.model_type for model_type in MODELS_WITH_POS_EMBEDDING_LIMIT)
```
This will be more generic as some Llama models may not have "llama" in their name.

Reply:
Okay. There are also some llama models which have a different model_type like mllama or mllama_text_model. Can we run those via text-generation?

![image](https://github.com/user-attachments/assets/d43e2d0f-dec9-4542-87fa-3588ef530d27)
![image](https://github.com/user-attachments/assets/1b29af4d-feed-47f4-9ec5-d515fbe237d5)

Reply:
Nope, mllama can be used with the image-to-text example: https://github.com/huggingface/optimum-habana/tree/main/examples/image-to-text

Reply:
Okay then it makes sense to use config.model_type. I will commit your suggestion. Thanks!